samza-kafka/src/test/java/org/apache/samza/system/kafka/TestKafkaSystemAdminJava.java

```diff
@@ -121,14 +124,13 @@ public void testValidateChangelogStreamDelegatesToValidateStream() {
 
     KafkaSystemAdmin systemAdmin = createSystemAdmin(coordProps, 3, Util.javaMapAsScalaMap(changeLogMap));
     SystemAdmin admin = Mockito.spy(systemAdmin);
-    StreamSpec spec = new StreamSpec("testId", STREAM, "testSystem", 12);
+    StreamSpec spec = new StreamSpec(StreamSpec.CHANGELOG_STREAM_ID, STREAM, "testSystem", 12);
 
-    admin.createChangelogStream(spec.getPhysicalName(), spec.getPartitionCount());
+    admin.createStream(spec);
     admin.validateStream(spec);
-    admin.validateChangelogStream(spec.getPhysicalName(), spec.getPartitionCount());
 
     Mockito.verify(admin).createStream(Mockito.any());
-    Mockito.verify(admin, Mockito.times(3)).validateStream(Mockito.any());
+    Mockito.verify(admin, Mockito.times(1)).validateStream(Mockito.any());
   }
 
   @Test
```

Comment:
Doesn't seem like this test is needed anymore. There's no delegation.

Reply:
Removed the test.